packages/sidecar/src/main.ts

```diff
@@ -179,7 +179,7 @@ const streamRequestHandler = (buffer: MessageBuffer<Payload>, incomingPayload?:
         }
 
         if (process.env.SPOTLIGHT_CAPTURE || incomingPayload) {
-          const timestamp = new Date().getTime();
+          const timestamp = BigInt(Date.now()) * 1_000_000n + (process.hrtime.bigint() % 1_000_000n);
```

Comment:
I'd love to use [uuidv7](https://uuid7.com/) for this but what you do works and is dependency-free so 🤷🏻‍♂️

Reply:
it also keeps the files chronologically sorted